File: payroll-api/src/main/java/com/company/payroll/model/Title.java
package com.company.payroll.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.RequiredArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@RequiredArgsConstructor
public class Title {
    private Integer titleNo;
    private String titleName;
    private String titleDesc;
}

File: kotlin-payroll/src/main/kotlin/com/company/payroll/model/Title.kt
package com.company.payroll.model

import com.company.payroll.annotation.NoArg
import lombok.AllArgsConstructor
import lombok.RequiredArgsConstructor

@NoArg
data class Title(
    var titleNo: Int,
    var titleName: String = "",
    var titleDesc: String = ""
)
